src/System.Data.Common/ref/project.json

```diff
@@ -1,14 +1,51 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Threading.Tasks": "4.0.0",
-    "System.IO": "4.0.0"
+    "Microsoft.Win32.Primitives": "4.3.0-beta-24604-02",
```

Comment:
> We will either need to add them as well or remove the dependent APIs.

@weshaggard, removing System.Transactions would mean removing this method:
```C#
public abstract partial class DbConnection : System.ComponentModel.Component, System.Data.IDbConnection, System.IDisposable
{
    public virtual void EnlistTransaction(System.Transactions.Transaction transaction);
}
```

To be clear, though, you're not actually talking about removing it from this ref, right?  You're instead talking about not having it in netstandard when the rest of these APIs are added there?  My understanding is there's nothing wrong with having additional members here, and this one seems fairly important to expose, at least in core.